tensorflow/compiler/xla/service/gpu/tests/gpu_kernel_tiling_test.cc

```diff
@@ -684,9 +684,9 @@ ENTRY main {
       ParseAndReturnVerifiedModule(kHloString, ConfigWithoutLayoutAssignment())
           .value();
   auto expected_ir = R"(
-; CHECK-LABEL: define void @fusion
+; CHECK-LABEL: define KERNEL_ANNOTATION @fusion
 ; CHECK: load <4 x i16>
-; CHECK-COUNT-4: load float
```

Comment:
load float is not generated on some AMDGPU archs so suggest removing this to avoid the _is_built_for_rocm conditional.